### PR TITLE
Ng/tic tac toe in react

### PR DESCRIPTION
**Pull request by NicollG**
Done with the quality review for the following course: https://egghead.io/courses/tic-tac-toe-in-react-387f

### CODE REVIEW

File: tic-tac-toe-in-react/02-create-our-game-grid-with-css-grid.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+I have cloned the repository from the Instructor and compared the dependencies that he is working with and make sure all of those dependencies are in the new project, however, this was not sufficient to make it work. 
+
+One of the dependencies that the instructor's repository works is => "parcel-bundler" This package has been deprecated [here](https://www.npmjs.com/package/parcel-bundler). Moreover, the repository works with the dependencies of "Babel' and the newest versions of React have those files hidden since they are already included when the app is created, for more information of this refer to this documentation [here](https://reactjs.org/docs/create-a-new-react-app.html). However, I download all of the Babel dependencies but not luck when running the app. 
+
+Finally, my last option was to upgrade the dependencies on the current project, specially React dependency in the repository, running the following command `yarn upgrade --latest react-scripts`. Doing that the project crashes. 
+
+I have spent more than one hour trying to figure out this problem. 
+
+From now on I'm reviewing the course and taking notes while working on the repository made by the instructor.
```
Comment (Creeland) [inline tic-tac-toe-in-react/02-create-our-game-grid-with-css-grid.md:59]: Good call to move on at that point.

File: tic-tac-toe-in-react/03-make-tic-tac-toe-game-buttons-interactive.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+</TimeStamp>
+
+<TimeStamp start="1:45" end="1:52">
+
+For more information about  lifting of state and state updaters in React review the following documentation [here](https://reactjs.org/docs/lifting-state-up.html#lifting-state-up)
+
+</TimeStamp>
+
+<TimeStamp start="2:55" end="3:05">
+
```
Comment (Creeland) [inline tic-tac-toe-in-react/03-make-tic-tac-toe-game-buttons-interactive.md:34]: I would try to break these long notes down into smaller chunks
Screen Shot 2021-06-30 at 2 36 08 PM

The goals is for them to look good in this format

**Review by Creeland — CHANGES_REQUESTED** [review]
The notes look good. I appreciate the detail you used when explaining what went wrong. It will save us time. 

I would just try to break up some of the longer notes into several smaller ones. Or just remove the code at the end of the lesson notes entirely

File: tic-tac-toe-in-react/03-make-tic-tac-toe-game-buttons-interactive.md (hunk 2 above)
Comment (NicollG) [reply tic-tac-toe-in-react/03-make-tic-tac-toe-game-buttons-interactive.md:34]: I have removed the pieces of code at the end of each lesson, this course is very practical, hence there is not a lot of room for notes.

**Review by Creeland — APPROVED** [review]

**Pull request merged**